index.php

```diff
@@ -44,8 +44,8 @@
 				endwhile;
 
 				the_posts_pagination( array(
-					'prev_text' => '<span class="screen-reader-text">' . __( 'Previous', 'twentyseventeen' ) . '</span>',
-					'next_text' => '<span class="screen-reader-text">' . __( 'Next', 'twentyseventeen' ) . '</span>',
+					'prev_text' =>  twentyseventeen_get_svg( $args = array( 'icon' => 'previous' ) ) . '<span class="screen-reader-text">' . __( 'Previous', 'twentyseventeen' ) . '</span>',
```

Comment:
Looks like this is also in

- `index.php` (this file)
- `archive.php`
- `comments.php`
- `single.php`
- `components/post/content.php`
- `inc/template-tags.php`